Dockerfile

```
@@ -33,7 +33,9 @@ COPY test/chai-as-promised.js /grist/test/chai-as-promised.js
 COPY app /grist/app
 COPY stubs /grist/stubs
 COPY buildtools /grist/buildtools
+COPY static/locales /grist/static/locales
 RUN yarn run build:prod
+RUN rm -rf /grist/static/locales
```

Hmm. The error seems to come from a new sanitization step applied to the locales, as @fflorent noted here https://github.com/gristlabs/grist-core/issues/1361#issuecomment-2572975295. If the sanitization step has an effect, does that effect get propagated to the final image? Reading this, I'm worried an unsanitized version of the locale files ends up in the final image.

Would like @berhalak and @georgegevoian's opinions, they should be back from breaks tomorrow and Wednesday respectively.

(Side note: I wonder if something that needs sanitization is found, the build should break rather than try to continue? Not sure if this new step is there to support some desired syntax or to avoid some undesired syntax. If the latter, failing loudly might be the right thing to do.)

---

I added this line to avoid the 151 line error:

```
# Finalize static directory
RUN \
  mv /grist/static-built/* /grist/static && \
  rmdir /grist/static-built
```

It has error: 

```
=> ERROR [stage-5 23/25] RUN   mv /grist/static-built/* /grist/static &&   rmdir /grist/static-built
------
 > [stage-5 23/25] RUN   mv /grist/static-built/* /grist/static &&   rmdir /grist/static-built:
0.443 mv: inter-device move failed: '/grist/static-built/locales' to '/grist/static/locales'; unable to remove target: Directory not empty
```

I think there is a problem in the steps order: locales are not sanitized at the right time in docker image build.

---

@tristanrobert that makes sense. @berhalak has tweaked the script to just fail if the translations are detected to contain dodgy material, and included the Dockerfile update. Thanks for your help with this!

---

Code is finally included in #1367 so it could be closed.